scripts/checks/verify_supply_chain.py

```
@@ -166,16 +156,61 @@ def verify_workflow_coverage() -> list[str]:
             for token in ["develop", "main", "push", "schedule", "ossf-scorecard"]
             if token not in scorecard
         )
+        if "main" in scorecard and "ossf/scorecard-action" in scorecard:
+            if "github.event.repository.default_branch" not in scorecard:
```

Ruff (0.15.12)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)